src/languages/bigquery/bigquery.formatter.ts

```diff
@@ -33,57 +33,113 @@ const reservedCommands = [
   // DDL, https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language
   'SET SCHEMA', // added
```

Comment:
Btw, I couldn't find SET SCHEMA anywhere in https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language.

Reply:
Indeed, looks like it's here by mistake and can be removed.

Reply:
I dropped it by myself.